Crashlytics/README.md

```diff
@@ -22,6 +22,16 @@ OR these two commands can be combined with
 You're now in an Xcode workspace generate for building, debugging and
 testing the FirebaseCrashlytics CocoaPod.
 
+### Updating crash report proto
+
+If the crash report proto needs to be updated, follow these instructions:
```

Comment:
It might be useful to give prereq installation instructions, specifically

gem (update|install) cocoapods cocoapods-generate
brew install protobuf nanopb
easy_install protobuf python